### ISSUE TITLE
train set performance

### ISSUE THREAD

**Issue by troublemeeter** [Cartus/AGGCN_TACRED#17]
hi, I would like to ask why the model on train set can only reach about 0.84 f1-value, does it means the model is under-fitting ? thanks!

**Comment by Cartus** [comment]
To be honest, I have never looked into the f1-value on the training set. It might be "under-fitting" the training set to some extent. However, what we really want to do is to achieve a better result to test set (based on the scores on the dev set). Probably you can tune the hyper-parameters to make the model fit the training set better (increase hidden sizes and layers, decrease dropout, etc,.), but I am not sure if you will get it better results on the test set. It is a trade-off.

**Comment by troublemeeter** [comment]
Thanks a lot, I will try.
